bots/tests-trigger

```diff
@@ -7,18 +7,6 @@ sys.dont_write_bytecode = True
 
 import github
 
-def trigger_image(api, opts):
```

Comment:
The github-trigger → tests-trigger rename needs to be reflected in bots/README.md.